gallery.sh

```diff
@@ -14,3 +14,5 @@ function iseek() {
     iseek
 }
 
+# icherry: Interactive staging of selected changed files (faster than git add --patch)
+alias icherry='git status -s | sed s/^...// | ipt -m | git add'
```

Comment:
hi @raiseandfall good idea! :smile: I was wondering why `icherry` as the command name and not `iadd` or something similar? I found it rather confusing as I was expecting `icherry` to be for actually cherry-picking something.

Here the only problem is that you have to convert data from the standard output into an argument at the end, to do so you would need to add `xargs` before `git add`